docs/reference/nuget-config-file.md

```diff
@@ -308,5 +340,19 @@ Below is an example `nuget.config` file that illustrates a number of settings:
     <apikeys>
         <add key="https://MyRepo/ES/api/v2/package" value="encrypted_api_key" />
     </apikeys>
+
+    <!--
```

Comment:
Do we need XML Comments in the docs?

Reply:
This is just following the same schema that the previous sections in this example:

https://github.com/NuGet/docs.microsoft.com-nuget/pull/1156/files#diff-a0e3252e3f37fca1ac18e064c0032269R335